src/App.js

```diff
@@ -117,6 +119,9 @@ class App extends Component {
           {this.renderListItem('Text Field', `${imagePath}/form_field_180px.svg`, 'text-field')}
           {this.renderListItem('Top App Bar', `${imagePath}/top_app_bar_180px.svg`, 'top-app-bar')}
         </ul>
+        <ul className='mdc-image-list standard-image-list mdc-top-app-bar--fixed-adjust'>
```

Comment:
We should put a title here and in the top, like "Components" and "Style", where "Style" would include things like color and typography (and possibly ripple?)

Maybe get design to look over this since it's not spec'd out. Open to names better than "Style" (Styling? Style Customization?)